src/sentry/utils/tag_normalization.py

```diff
@@ -87,7 +87,7 @@ def normalize_sdk_tag(tag: str) -> str:
         return "other"
 
     # collapse tags other than JavaScript / Native to their top-level SDK
-
+    # TODO @athena: Remove comment before merging
```

Comment:
This is a tag, so I'm thinking to keep it as is and not modify. Please correct me if I'm wrong.